site/content/entry/compatability-list-wiki-created.md

```diff
@@ -0,0 +1,15 @@
++++
+date = "2017-06-03T11:30:00-04:00"
+title = "Citra Now Has a Games Compatibility Wiki!"
+author = "Saphiresurf"
++++
+
+
+As game compatibility has broadened and the popularity of Citra has risen there's always been a lack of consistent information on what could be played in Citra and what couldn't. Often times the best way to find out if a game was working properly in Citra was to jump in and test it for yourself, but if any issues were encountered you'd be left scouring through forums sifting through potentially out of date information for answers.
+
+In order to address this issue Flame Sage, Flamboyant Ham, Leo121, Anodium, and Koopa have worked hard to create a new game compatibility wiki! As time goes on more and more titles from the 3DS library will be added to this list. As they're added in, each game will be given a compatibility rating based on how well and how smoothly they run in Citra. On each games individual page there will be a summary of the game itself, how it runs on Citra, as well as any problems that arise when running the game. With all of this information it works to give you a much better idea of how it runs and what to expect.
```

Comment:
Attribution is good, but it'd be best if you directly linked to their GitHub profiles, and it'd be best if you preserved case, since many websites and services are case-sensitive:

```diff
--- In order to address this issue Flame Sage, Flamboyant Ham, Leo121, Anodium, and Koopa have worked hard to create a new game compatibility wiki!
+++ In order to address this issue [chris062689](https://github.com/chris062689), [Schplee](https://github.com/Schplee), [leo121](https://github.com/leo121), [anodium](https://github.com/anodium), and [TheKoopaKingdom](https://github.com/TheKoopaKingdom) have worked hard to create a new game compatibility wiki!
```